components/pedigree/resources/src/main/resources/pedigree/view/person.js

```diff
@@ -1579,6 +1579,8 @@ define([
                     //          {id: 'ENSG00000160688', gene: 'FLAD1', status: 'solved'},
                     //          {id: 'custom', 'gene': 'custom', status: 'candidate'} ]
                     this._setGenes(info.genes);
+                } else {
+                    this._setGenes([]);
```

Comment:
agree, probably it would be better. Just as many things this started form 3 or 4 fields which were different enough that coming up with a shared code was more pain than do it this way, and then it grew and has multiple similar entries. We should probably make a PR to refactor data/data field handling in pedigree, but not for this PR/not for 1.3